### ISSUE TITLE
Request for some IPA and rarer characters

### ISSUE THREAD

**Issue by theblursed** [CatharsisFonts/Ysabeau#58]
Dear CatharsisFonts,

In thinking about a new linguistics project, I’m again considering your Ysabeau as a possibility for the main typeface. I want to compliment you again for the work of beauty that Ysabeau is.

Unfortunately, for the use of linguistics today, it is difficult to avoid using the International Phonetic Alphabet; in the case of Ysabeau, the lack of even the main IPA characters is particularly painful, as the typeface is otherwise very rich in diacritics and rare characters, and for its clarity, richness and beauty of design, it would otherwise be a natural choice in many contexts.

I'm therefore writing you to ask for the addition of some characters that I'd find particularly useful; and, as I'm already here, also some rarer, non-IPA ones, that I used in some works and may use again (and/or in new editions of old works).

— _Main needs_
Understanding very well that the whole IPA character set is no small feat nor small work for anybody, I’d ask you only for the addition of the characters I use mostly:
`ɛ` `ɔ` `ɲ` `ʦ` `ʣ` `ʧ` `ʤ` `ʃ` `ʎ` `ɥ`

I’d also need the ` ̍` (_combining vertical line above_) to correctly combine with the following vowels, and `V` to denote a generic vowel:
`a̍` `e̍` `i̍` `o̍` `u̍` `ɛ̍` `ɔ̍` `y̍` `V̍`

— _Would be nice_
(In IPA transcriptions, for legibility purposes it would be useful to have the top terminal of `f`, `ʃ` and `ʧ` not touch the `i̍` in `fi̍`, `ʃi̍` , `ʧi̍`).

I’d like `ʃ` to be usable also beyond IPA purposes; it would be nice if it could have ligatures and auto-adjusting top length like `ſ` in `ʃi`, `ʃj`, `ʃí`, `ʃï`.

(Ligatures for `ʧi` and `ʧj`, as are found in Brill and Junicode, would be a refined esthetic touch, but are not strictly necessary).

— _Less necessary_
It would also be nice if the new IPA vowels (`ɛ`, `ɔ`) could combine smoothly with other diacritics (well, for now I may need only `´`: `ɔ́`, `ɛ́`).

Other less common characters that I may find useful to have:
`ʋ` `Ʋ` `Ɔ` `Ɛ` `Ʃ` `ᘕ`
`Ɔ` and `Ɛ` also with the possibility of combining with  `´`: `Ɔ́`, `Ɛ́`.

Sorry for the many things, and thank you for anything you'll decide to do! Have a good day. 🙂

**Comment by CatharsisFonts** [comment]
Dear Blursed,

as a conlanger, I have a special place in my heart for IPA, and I’ve always wanted to add it to Ysabeau in particular. But as you said, the task is dauntingly gargantuan, and comes with a slew of obscure difficulties. I currently don’t have time for it. If you were to convince Dave Crossland (or, in fact, any institution of your choice) to fund me for the project, though, I’d be all over it. 🤓

Cheers, Christian

**Comment by theblursed** [comment]
> as a conlanger [...]

I had no idea, that is very interesting to know. I’ve been [surprised by how much I've become] interested in artificial languages for some years now, but I do not follow a lot the current “scene” (mainly because of lack of time... me too). I see in the Internet you’re a _professional_ conlanger? That’s neat! 😄

> I have a special place in my heart for IPA, and I’ve always wanted to add it to Ysabeau in particular. But [...] I currently don’t have time for it. If you were to convince Dave Crossland (or, in fact, any institution of your choice) to fund me for the project, though, I’d be all over it.

Understandable. Well, I can’t say I’m good at raising money, unfortunately… but “there’s no harm in trying”, as we say in my country. I'll see if I can do something. “It doesn't happen, but if it happens...”

**Comment by CatharsisFonts** [comment]
Dear @theblursed, 

I've started implenting IPA in Ysabeau. Currently I'm supporting all glyphs in the IPA tab in Glyphs App, but I've noticed it doesn't cover all of the glyphs in ipa.type.it (especially some of the superscript glyphs are missing). Also, currently only Roman cuts from Hairline to Bold are implemented. I intend to fill out ipa.type.it and the remaining masters, but probably not the «less common characters» you mention. 

Among the things you requested, the vertical accent seems to work fine in Font Book. The esh doesn't seem to need any ligatures because of its reclining base shape.

Does this work for you?

Cheers, Christian

<img width="960" alt="Screenshot 2024-11-04 at 08 39 51" src="https://github.com/user-attachments/assets/7526d3d7-e6e7-42fa-8007-03c6783d0873">

**Comment by theblursed** [comment]
Dear Christian,

Thank you very much!

> Among the things you requested, the vertical accent seems to work fine in Font Book.

Perfect. 👌

> The esh doesn't seem to need any ligatures because of its reclining base shape.

I see; I was thinking about the commoner vertical stem shape, which I usually prefer aesthetically (for coherence with similar letters like `f`, `ſ` and `j`): but obviously the choice is yours as the designer.
![Schermata 2024-11-05 alle 11 04 15](https://github.com/user-attachments/assets/94e6592a-c850-4e15-abc6-097d53c7da9e)
(A rare design choice I recently found "strangely beautiful" is used in Palemonas MUFI: vertical stem and asymmetrical curls. It works pretty well in text in my opinion)
![Schermata 2024-11-05 alle 11 04 59](https://github.com/user-attachments/assets/7cdf25a4-7df6-4416-8649-811bd7935bba)
> I intend to fill out ipa.type.it and the remaining masters, but probably not the «less common characters» you mention.

I don’t think I'll be needing those soon, but maybe take them in consideration for some future expansion: among other things they'd make it easier to write some African languages, that use (some) IPA letters also in uppercase ([for example](https://en.wikipedia.org/wiki/Latin_epsilon#List_of_languages_that_use_Latin_epsilon) for `Ɛ`).

Again, thank you for this beautiful beginning-of-the-week surprise 🤗

**Comment by CatharsisFonts** [comment]
I agree that a more vertical esh would be somewhat more aesthetically pleasing, but I don't want to go through the headache of assuring that it plays well with preceding descenders and subsequent ascenders. Also, I'm following in the footsteps of EB Garamond, which has this swashy sigmoid esh as well.

**Comment by theblursed** [comment]
> [...] I'm following in the footsteps of EB Garamond, which has this swashy sigmoid esh as well.

Since you brought it up, I must confess (as a funny apology) that around one year ago I self-published a booklet written in a proposed new orthography for Italian (I’m Italian), that uses esh for /z/: I really like EB Garamond and used it, but… I thought its esh was not beautifully consistent with the rest, so, even without knowing pretty much anything about creating digital types, I roughly made up a new esh glyph based on Junicode's esh (along with personally [😅] drawn not-very-beautiful `ʃi` and `ʃj` ligatures)… yes, I’m that kind of madman.

![Schermata 2024-11-05 alle 23 59 55](https://github.com/user-attachments/assets/09f0ab22-763f-4796-a6c9-1f051b993b80)
![Schermata 2024-11-06 alle 00 00 31](https://github.com/user-attachments/assets/f87163e7-e545-4d5f-8009-9c5caf7cc242)

So don’t worry for my preferences! I’m very nitpicky, if there’s a fault it’s on me.

> I agree that a more vertical esh would be somewhat more aesthetically pleasing, but I don't want to go through the headache [...]

Completely understandable. I know refined details take a great deal of work and time. Having a full functioning IPA in Ysabeau is already a beautiful gift for me and everyone else. Thank you again for your admirable work and your kindness. 🤗
